src/sentry/lang/javascript/processor.py

```diff
@@ -62,6 +62,9 @@
 )
 VERSION_RE = re.compile(r"^[a-f0-9]{32}|[a-f0-9]{40}$", re.I)
 NODE_MODULES_RE = re.compile(r"\bnode_modules/")
+# Default Webpack output path using multiple namespace - https://webpack.js.org/configuration/output/#outputdevtoolmodulefilenametemplate
+# eg. webpack://myproject/./src/lib/hellothere.js
+WEBPACK_NAMESPACE_RE = re.compile(r"^webpack://[a-zA-Z0-9_\-@\.]+/\./")
```

Comment:
Couldn't find any tests in the webpack repo around this, so I'm just going to assume this regex should be good enough.